waves/wallet/src/registry.rs

```diff
@@ -0,0 +1,13 @@
+use elements_fun::AssetId;
+use std::collections::HashMap;
+
+pub fn registry() -> HashMap<AssetId, (String, u8)> {
```

Comment:
Could this function maybe be `const`?

You might need to change the `String` for `&'static str`.

Reply:
Also, it might be better to expose a lookup function from this module instead of the hashmap (which is kind of an implementation detail).